types/serverless/serverless-tests.ts

```diff
@@ -128,11 +128,27 @@ getHttp(
     {
         http: {
             path: 'myPath',
+            // @ts-expect-error
+            mehtod: 'get',
             method: 'get',
         },
     },
     'myFunction',
 );
+
+// @ts-expect-error - Test missing properties
+getHttp({}, 'myFunction');
+
+// @ts-expect-error - Test invalid properties
+getHttp({}, 'myFunction', { invalid: 'property' });
+
+getHttp({
```

Comment:
you should add a test of the return type for getHttp. Dropping the type inference is probably fine but means, technically, that if you save the result in a variable `x` then `x.method` will be available now, but not in the old code.

Reply:
Very good point, will do soon :). Thanks a lot!